screens/Notifications.js

```diff
@@ -1,19 +1,35 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image, Dimensions } from 'react-native';
+import { StyleSheet, Text, View, Image, Dimensions, FlatList } from 'react-native';
 
 
 export default class Notifications extends React.Component {
   render() {
     return (
       <View style={styles.container}>
-        <View style={styles.orangeRectangle}></View>
+        <View style={styles.peachBox}></View>
         <Text style={styles.header}>Notifications!</Text>
-        <View style={styles.whiteCard}></View>
-        <View style={styles.slantedBlueRectangle}></View>
-        <View style={styles.slantedPurpleRectangle}></View>
+        <View style={styles.iconsBox}>
+          <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.shopCart} />
+          <Image source={require('./../assets/images/profile_head.png')}style={styles.personIcon} />
+        </View>
+        <View style={styles.blueBox}></View>
```

Comment:
`View` components that don't have content inside them, can just be closed like `<View />`